collectors/cgroups.plugin/cgroup-network-helper.sh

```diff
@@ -184,6 +186,39 @@ virsh_find_all_interfaces_for_cgroup() {
     fi
 }
 
+# -----------------------------------------------------------------------------
+# netnsid detected interfaces
+
+netnsid_find_all_interfaces_for_pid() {
+    local pid="${1}"
+    [ -z "${pid}" ] && return 1
+
+    local nsid=$(lsns -t net -p ${pid} -o NETNSID -nr)
+    [ -z "${nsid}" -o "${nsid}" = "unassigned" ] && return 1
+
+    set_source "netnsid"
+    ip link show |\
+        grep -B 1 -E " link-netnsid ${nsid}($| )" |\
+        ip link show | grep -B 1 -E " link-netnsid ${nsid}($| )" | sed -n -e "s|^[[:space:]]*[0-9]\+:[[:space:]]\+\([A-Za-z0-9_]\+\)\(@[A-Za-z0-9_]\+\)*:[[:space:]].*$|\1|p"
```

Comment:
why is this piping `ip link` to `grep` to `ip link` to `grep`?

Reply:
oops! just merged it!
This is wrong of course. Making a new PR...

Reply:
https://github.com/netdata/netdata/pull/5417